### PR TITLE
sbcl 2.4.3

### PR DESCRIPTION
**Pull request by Porkepix**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by timsutton — APPROVED** [review]

**Comment by p-linnane** [comment]
The runners are still active on the back end, but agreed they should've been done ages ago. Will force push to restart.

**Comment by p-linnane** [comment]
I tried building this locally to see what's going on. `acl2` Is waiting for input:
```
╰─ brew install -sv acl2
==> Fetching acl2
==> Downloading https://github.com/acl2/acl2/archive/refs/tags/8.5.tar.gz
==> Downloading from https://codeload.github.com/acl2/acl2/tar.gz/refs/tags/8.5
     #     #        #          #                                     -=O=-
/usr/bin/env tar --extract --no-same-owner --file /Users/patrick/Library/Caches/Homebrew/downloads/1d966716dafa161d7665b8c8359134d1a1f56d4e7efee895b104526b8e486640--acl2-8.5.tar.gz --directory /private/tmp/homebrew-unpack20240229-91908-815md4
/usr/bin/env cp -pR /private/tmp/homebrew-unpack20240229-91908-815md4/acl2-8.5/. /private/tmp/acl2-20240229-91908-fnitf/acl2-8.5
==> make LISP=/opt/homebrew/bin/sbcl ACL2=/private/tmp/acl2-20240229-91908-fnitf/acl2-8.5/saved_acl2 USE_QUICKLISP=0 all basic
Preparing to build saved_acl2 (log file make.log).

Type HELP for debugger help, or (SB-EXT:EXIT) to exit from SBCL.

restarts (invokable by number or by possibly-abbreviated name):
  0: [RETRY   ] Retry EVAL of current toplevel form.
  1: [CONTINUE] Ignore error and continue loading file "/private/tmp/acl2-20240229-91908-fnitf/acl2-8.5/acl2-check.lisp".
  2: [ABORT   ] Abort loading file "/private/tmp/acl2-20240229-91908-fnitf/acl2-8.5/acl2-check.lisp".
  3:            Exit debugger, returning to top level.

((LAMBDA NIL :IN "/private/tmp/acl2-20240229-91908-fnitf/acl2-8.5/acl2-check.lisp"))
   source: (ERROR "This Common Lisp is unsuitable for ACL2 because the ~
                       character~%with char-code ~d ~a standard in this ~
                       Common Lisp but should~%~abe."
                  I
                  (IF (STANDARD-CHAR-P CH)
                      "is"
                      "is not")
                  (IF (STANDARD-CHAR-P CH)
                      "not "
                      ""))
0]
```

**Comment by justjoheinz** [comment]
I do not quite understand what maxima and acl2 have to do with sbcl. Could the sbcl part be split into a separate PR?

**Comment by p-linnane** [comment]
> I do not quite understand what maxima and acl2 have to do with sbcl. Could the sbcl part be split into a separate PR?

They depend on `sbcl`. We test dependents when upgrading formulae so that we don't introduce breakage.

**Comment by timsutton** [comment]
Just repro-ed this. The issue doesn't happen with sbcl `--head`, and according to https://www.sbcl.org/news.html there's usually a new release at the end of each month. So if nothing else, this should be able to be superseded by a new release in a couple weeks.

**Comment by chenrui333** [comment]
they just made a new release, 2.4.3, I guess we can try in this PR to see how it goes.

File: Formula/s/sbcl.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Sbcl < Formula
   desc "Steel Bank Common Lisp system"
   homepage "http://www.sbcl.org/"
-  url "https://downloads.sourceforge.net/project/sbcl/sbcl/2.4.1/sbcl-2.4.1-source.tar.bz2"
-  sha256 "da4f9486fad413d3af7abb1b09269325fdb4bff7e7b88f2195ddee743263cf7e"
+  url "https://downloads.sourceforge.net/project/sbcl/sbcl/2.4.2/sbcl-2.4.2-source.tar.bz2"
```
Comment (SMillerDev) [inline Formula/s/sbcl.rb:4]:
```suggestion
  url "https://downloads.sourceforge.net/project/sbcl/sbcl/2.4.3/sbcl-2.4.3-source.tar.bz2"
```

**Review by iMichka — APPROVED** [review]

**Pull request merged**